### PR TITLE
Organize cards

### PR DESCRIPTION
**Pull request by mcmullinboy15**
I've added a v-card in between the cards and the 'Draw from Deck' v-card. This v-card gives the player the ability to organize their hand by Color or Number.

### CODE REVIEW

**Comment by jak103** [comment]
@mcmullinboy15 Can you fix the merge issues so we can get this in the deployed game?

File: client/src/views/About.vue [hunk 1]
```diff
@@ -128,12 +151,46 @@ export default {
...
+        this.loadingHand = true;
+      }
+      this.sortByNum = false;
+      this.sortByColor = false;
+    },
+    orgByNum() {
+      this.sortByNum = true;
+      this.sortByColor = false;
+
+      var colors = { 'red': 0, 'blue': 1 , 'green': 2, 'yellow': 3 }
```
Comment (Aaron-Fine) [inline client/src/views/About.vue:166]: colors can probably be const here

File: client/src/views/About.vue [hunk 2]
```diff
@@ -128,12 +151,46 @@ export default {
...
+      this.cards.sort((a, b) => { return colors[a.color] - colors[b.color];});
+
+      return this.cards.sort((a, b) => { return a.number - b.number;});
```
Comment (Aaron-Fine) [inline client/src/views/About.vue:169]: Shouldn't this be `return a.number < b.number` both here and in the other sorts?

File: client/src/views/About.vue [hunk 3]
```diff
@@ -56,6 +56,18 @@
           <v-card v-else-if="!!game_over">{{game_over}} has won the game!</v-card>
 
           <v-card v-else :class="'ma-3 pa-6'" outlined tile>Waiting for {{ current_player }}</v-card>
+
+          <!-- Organize Cards -->
+          <v-card v-if="username == current_player" :class="'ma-3 pl-6 pa-4'" outlined tile>
```
Comment (Aaron-Fine) [inline client/src/views/About.vue:61]: Can you include a screenshot showing these changes?

Comment (mcmullinboy15) [reply client/src/views/About.vue:61]: image

File: client/src/views/About.vue (hunk 2 above)
Comment (mcmullinboy15) [reply client/src/views/About.vue:167]: That is a valid argument, haha! Idk, I just follow StackOverFlow sometimes and if it works perfectly then I don't fight it. haha

File: client/src/views/About.vue (hunk 1 above)
Comment (mcmullinboy15) [reply client/src/views/About.vue:164]: I'm sure I could but I don't know if it matters. And this way I can decide the order I want.

**Comment by mcmullinboy15** [comment]
@jak103  I fixed the Merge Conflicts

**Pull request merged**